docs/DeveloperGuide.md

```diff
@@ -1,10 +1,54 @@
 # Developer Guide
 
-## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
-## Design & implementation
+# Acknowledgements
+Tantou's structure has been greatly inspired by the developer's respective iPs as listed below:
+1. [Donovan](https://github.com/xenthm/ip)
+2. [Sarah](https://github.com/sarahchow03/ip)
+3. [Ian](https://github.com/iaso1774/ip)
+4. [Andy](https://github.com/averageandyyy/ip)
+
+# Design & implementation
+
+{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Overall Architecture
+### Commands Overview
+![Command Inheritance](/docs/uml/images/Command.png)
+The current list of viable `Commands` are as follows:
```

Comment:
I like how you included `Overall Architecture` as a section. It helps to provide a high-level view of our code first.